### PR TITLE
add logger for plug, update example

### PR DESCRIPTION
**Pull request by wallacbe**

### CODE REVIEW

File: openhtf/plugs/__init__.py [hunk 1]
```diff
@@ -115,6 +117,9 @@ class InvalidPlugError(Exception):
 
 class BasePlug(object): # pylint: disable=too-few-public-methods
   """All plug types must subclass this type."""
+  
+  def __init__(self):
+    self.logger = logging.getLogger(logs.RECORD_LOGGER)
```
Comment (fahhem) [inline openhtf/plugs/__init__.py]: if you put this at the class level, it should still work and won't require a `super()` call

Comment (grybmadsci) [inline openhtf/plugs/__init__.py]: hm, as another option, if you do something like this, you get plug type info built into the logger name:
```
@property
def logger(self):
  return logging.getLogger('.'.join((logs.RECORD_LOGGER, 'plugs', type(self).__name__)))
```

**Comment by wallacbe** [comment]
Made changes, PTAL

**Comment by fahhem** [comment]
LGTM

**Pull request merged**